src/clients/clone_system.rb

```diff
@@ -23,11 +23,30 @@ def main
       Yast.import "Y2ModuleConfig"
       Yast.import "Mode"
       Yast.import "FileUtils"
+      Yast.import "Report"
+      Yast.import "Installation"
 
       textdomain "autoinst"
 
       @moduleList = ""
 
+      if Mode.normal
+        if !PackageSystem.Installed("autoyast2")
+          ret = PackageSystem.InstallAll(["autoyast2"])
+          # The modules/clients has to be reloaded. So the export
+          # will be restarted.
+          if ret
+            SCR.Execute(
+              path(".target.bash"),
+              "touch #{Installation.restart_file}"
+            )
+          end
+          return
+        else
+          SCR.Execute(path(".target.remove"), Installation.restart_file)
```

Comment:
I'm not sure if unconditional removal works, what happens if the file does not exist? (JFYI this code uses a check before removal: https://github.com/yast/yast-installation/blob/master/src/include/installation/inst_inc_second.rb#L427)